astroquery/nasa_exoplanet_archive/core.py

```diff
@@ -510,6 +586,30 @@ def _handle_all_columns_argument(self, **kwargs):
 
         return kwargs
 
+    @class_or_instance
+    def _request_to_sql(self, request_payload):
+        """Convert request_payload dict to SQL query string to be parsed by TAP."""
+
+        # Required minimum query string
+        query_req = "select {0} from {1}".format(request_payload.pop("select", "*"), request_payload.pop("table", None))
+        if "order" in request_payload.keys():
+            request_payload["order by"] = request_payload.pop("order")
+        if "format" in request_payload.keys():
+            responseformat = request_payload.pop("format")  # figure out what to do with the format keyword
+        if "ra" in request_payload.keys():  # means this is a `query_region` call
+            request_payload["where"] = "contains(point('icrs',ra,dec),circle('icrs',{0},{1},{2}))=1".format(request_payload["ra"], request_payload["dec"], request_payload["radius"])
+            del request_payload["ra"]
+            del request_payload["dec"]
+            del request_payload["radius"]
+        if "where" in request_payload:
+            if "pl_hostname" in request_payload["where"]:  # means this is a `query_object`
+                request_payload["where"] = "pl_hostname or pl_name like {0}".format(request_payload["where"][request_payload["where"].find("=")+2:request_payload["where"].find("OR")-2])  # This is a bit hacky since we are getting this from the request_payload (downstream) instead of directly from object_name
```

Comment:
could you make this fit onto one line?  and line 600 above

Reply:
Fixed.